configure.py

```diff
@@ -1132,6 +1142,16 @@ def configure_napi(output):
   version = getnapibuildversion.get_napi_version()
   output['variables']['napi_build_version'] = version
 
+  if options.arch_triplet:
+    output['variables']['arch_triplet'] = options.arch_triplet
+  else:
+    output['variables']['arch_triplet'] = 'unknown-unknown-unknown'
```

Comment:
`process.config.variables.arch_triplet` doesn't seem to be used anywhere?

Reply:
It is used by gyps and other package that need to know the architecture. Maybe need documentation but needed on debian/unbuntu context

Reply:
If it's not set in our release builds, it doesn't strike me as that useful for gyp / node-gyp.

When and where exactly would this be used? Build files normally inspect the `OS` and `target_arch` variables and they won't care about the ABI because that's just "whatever node uses."